src/components/ConnectButton/index.tsx

```diff
@@ -20,7 +19,6 @@ const wallets = getSupportedWallets()
 
 export const onboard = Onboard({
   networkId: parseInt(networkId, 10),
-  networkName,
```

Comment:
Just saw this removal. This is important so unknown networks for bnc-onboard doesn't show Unknown everywhere on their modal but the expected name. This can be tested on Volta for example

![imagen](https://user-images.githubusercontent.com/25051234/129561580-30c0a197-5ed8-4941-b0d4-c59b4bf4cbc9.png)

Reply:
Oops, will restore